### PR TITLE
Add about page that houses our roadmap, donation info, and FAQs.

### PR DESCRIPTION
**Pull request by JonathonDunford**
Add link in nav to about page, remove old help section.

### CODE REVIEW

File: about/index.html [hunk 1]
```diff
@@ -0,0 +1,472 @@
...
+				integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+				crossorigin="anonymous"></script>
+
+		<script>
+			
+			$(document).ready(function($) {
+
+				//if page loaded with a hash, smooth scroll to it
+				if (window.location.hash !== '') {
+					scroll_to_hash(window.location.hash);
```
Comment (freeatnet) [inline about/index.html]: Let's not mess with scroll behaviour.

File: about/index.html [hunk 2]
```diff
@@ -0,0 +1,472 @@
...
+				background: #77DD77;
+			}
+
+		</style>
+	</head>
+
+	<body>
+
+		<div class="header">
+			<a href="https://forkdelta.github.io">Trade</a>
```
Comment (freeatnet) [inline about/index.html]: This link doesn't seem to work:
```
(index):42 Uncaught TypeError: Cannot read property 'top' of undefined
    at scroll_to_hash ((index):42)
```

File: about/index.html [hunk 3]
```diff
@@ -0,0 +1,472 @@
...
+			.timeline .demo-card {
+				background: #394b59;
+			}
+
+			.timeline .demo-card.green {
+				background: #77DD77;
+			}
+
+		</style>
+	</head>
```
Comment (freeatnet) [inline about/index.html]: Let's add a title and some meta / OG tags.

File: about/index.html (hunk 1 above)
Comment (JonathonDunford) [reply about/index.html]: It's impossible to directly link to parts of the page otherwise.

File: about/index.html (hunk 2 above)
Comment (JonathonDunford) [reply about/index.html]: Fixed.

File: about/index.html (hunk 3 above)
Comment (JonathonDunford) [reply about/index.html]: Fixed.

File: about/index.html (hunk 1 above)
Comment (freeatnet) [reply about/index.html]: `a name` or `id` should work fine. Am I missing something?

Comment (JonathonDunford) [reply about/index.html]: It just looks a lot better when smoothed. Jumping around is clunky.

Comment (freeatnet) [reply about/index.html]: Which brings me back to: let's not mess with scrolls. If the user feels their OS / browser scroll behaviour is unacceptable, let them address that.

**Comment by freeatnet** [comment]
@JonathonDunford Looks like we need a rebase.

**Pull request merged**